ui/apps/platform/src/Containers/Clusters/ClustersTablePanel.js

```diff
@@ -207,7 +207,7 @@ function ClustersTablePanel({ selectedClusterId, setSelectedClusterId, searchOpt
     const headerComponent = (
         <TableHeader
             length={currentClusters?.length || 0}
-            type="Cluster"
+            type="cluster"
```

Comment:
Most of your changes in this PR have been going from all-lowercase (or all-uppercase) to "Sentence case".

This one is going from sentence to all-lowercase. Is that necessary?

Reply:
Yes, for 2 reasons:
* For table count, the number is the first word of the sentence and entity type is second word.
* The count links in Configuration Management and Vulnerability Management that people click to visit these pages have **1 cluster** as pattern.